cli/lib/metadata.ts

```diff
@@ -159,6 +158,7 @@ export async function injectMetadata(this: Context, jobUri: string) {
     }
 
     if (quad.predicate.equals(cube.observedBy)) {
+      const creatorTerms = maintainer.pointer.out(cube.observedBy).terms
       for (const creator of creatorTerms) {
         if (creator.termType === 'NamedNode') {
```

Comment:
Is this check needed?

Reply:
Other than satisfying the type system, technically a term can be multiple things and we do want an URI